### PR TITLE
chore(client): boilerplate code frontend 

### PR DESCRIPTION
**Pull request by gordonpn**
Next.js project with Chakra UI, ESLint, Prettier.
Husky pre-commit hook at the root of the project.
Frontend local dev Docker setup. [CU-k0uuva](https://app.clickup.com/t/k0uuva)

### CODE REVIEW

**Comment by Mackbellemore** [comment]
Task linked: [CU-jz28ct Setup front-end React.js](https://app.clickup.com/t/jz28ct)

File: client/.prettierrc.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (Mackbellemore) [inline client/.prettierrc.json:1]: can i vote to have tab spacing lol

File: client/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+version: "3.8"
```
Comment (Mackbellemore) [inline client/docker-compose.yml:1]: We could have just one compose file at the root of the project

File: client/.prettierrc.json (hunk 1 above)
Comment (gordonpn) [reply client/.prettierrc.json:1]: Done!

File: client/docker-compose.yml (hunk 2 above)
Comment (gordonpn) [reply client/docker-compose.yml:1]: Ultimately, we'll have just one docker-compose file when the backend project is up. We can delete this file when we revisit it in [CU-k0uuva](https://app.clickup.com/t/k0uuva)

Comment (Mackbellemore) [reply client/docker-compose.yml:1]: sounds good

**Review by Mackbellemore — APPROVED** [review]

**Review by Mackbellemore — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+## Getting started with development
+
+Clone the GitHub repository and create your own branch using the auto-generated ClickUp naming convention.
+
+### Making code changes
+
+Husky is set up at the root of the project.
+
+Husky will run Prettier before ever commit (pre-commit hook). This will ensure that we have consistent code style across all developers.
```
Comment (DavidLiang01) [inline README.md:11]: I think there is a typo here: 'ever' -> every

Comment (gordonpn) [reply README.md:11]: You are right

**Comment by DavidLiang01** [comment]
Should the default components that comes with the setup be removed? Such as CTA.js, container.js

**Comment by DavidLiang01** [comment]
Did we decide on which state management are we going to use? Was it MobX?

**Comment by Mackbellemore** [comment]
> Did we decide on which state management are we going to use? Was it MobX?

Ya we said MobX but were leaving that for another PR

**Comment by gordonpn** [comment]
> Should the default components that comes with the setup be removed? Such as CTA.js, container.js

I was thinking of leaving them as examples until we develop our own components then we can remove them.

> Did we decide on which state management are we going to use? Was it MobX?

I'm indifferent about the state management library, as long as it's not Redux. We can add the library and the boilerplate when we will need it. This PR is just for the basic setup.

**Comment by DavidLiang01** [comment]
Great!

**Review by DavidLiang01 — APPROVED** [review]

**Review by Mackbellemore — APPROVED** [review]

**Pull request merged**